t.cs

```diff
@@ -0,0 +1,23 @@
+public class Test
+{
+	public bool T(bool c)
+	{
+        if (c)
+		{
+			T(false);
+		}
```

Comment:
Closing brace should be followed by blank line.

```suggestion
		}

```